auth_otp/views_extended.py

```
@@ -0,0 +1,310 @@
+    try:
+        success = AuthService.revoke_session(request.user, session_id)
+        
+        if success:
+            return Response(
+                {
+                    'success': True,
+                    'message': 'نشست با موفقیت لغو شد'
+                },
+                status=status.HTTP_200_OK
+            )
+        else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)